code/datums/jobs.dm

```diff
@@ -138,6 +138,51 @@
 				M.choose_name(3, src.name, default)
 				if(M.real_name != default && M.real_name != orig_real)
 					phrase_log.log_phrase("name-[ckey(src.name)]", M.real_name, no_duplicates=TRUE)
+				if((M.job == "Clown") || (M.job == "Mime"))
+					var/make_bracelet = FALSE //friendship bracelets!!!
+					var/friend_limit = 4
+					for(var/mob/MOB in mobs)
+						if((MOB != M) && ((MOB.job == "Clown") || (MOB.job == "Mime")))
```

Comment:
Just thought of something else for this...should probably verify that the friend is alive (and maybe that they have a client)

also probably check for `hibernating = 1` - which is a var on `mob/living` set when they go into the cryotron

I can only imagine the sadness of a clown looking for a non-existent friend

that way you don't get bracelets for the dead or clowns in cryo

Reply:
done～❤️